frontend/playwright.config.ts

```diff
@@ -0,0 +1,67 @@
+import { defineConfig, devices } from "@playwright/test";
+
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+    testDir: "./tests",
+    workers: 1,
```

Comment:
> Why would one want to do that?

You mean restricting wokers on CI? According to https://playwright.dev/docs/ci#workers, it's to "prioritize stability and reproducibility".

> And why do we always restrict us to one worker?

The tests for `blocks` and `realms` create and rely on specific realms. When any of these run at the same time in chromium and firefox, that can lead to race conditions.
Unfortunately, it's not possible to configure worker usage for a single test file yet, as far as I can tell.

Hmm but I suppose I might be able to circumvent this by using different realms based on which browser the test is run in - that should be possible. Will look into that.

Reply:
Thanks for the explanation! 

And yeah, the race conditions seem kind of unfortunate. I mean... ideally every test would get its own fresh database and be completely isolated. But I'm sure setting this up is toooons of work. So if you find a way to avoid race conditions in a simply way as you suggest, why not go for it. That said, don't invest too much time into this. Running locally with 1 worker is certainly fine for now.

Reply:
So you changed it locally to `undefined`. So the race condition stuff is solved?